notebook2/Dockerfile

```diff
@@ -15,7 +15,9 @@ RUN apk add \
   flask \
   Flask_Sockets \
   kubernetes \
-  'urllib3<1.24'
+  'urllib3<1.24'\
+  libsass \
```

Comment:
ah yeah, so currently we don't test notebook/notebook2 automatically.

I usually test my PRs by creating a temporary k8s namespace, and modifying the yaml to deploy into said namespace, that way I can ensure it all starts up correctly. This also, of course, requires putting the necessary secrets and serviceaccounts into said namespace. The configuration for the service accounts are stored in `vdc/k8s-config.yaml`. For the secret, I manually regenerate it using `kubectl create secret generic`

Reply:
Got it, thanks!